recipe/meta.yaml

```diff
@@ -1,7 +1,7 @@
-{% set version = "1.10" %}
-{% set sha256 = "bcbe8ada8c435a253b1fa438f505dd311832a88f52316fd7d30342b106a4242c" %}
-{% set runc_version = "0.1.0" %}
-{% set runc_sha256 = "78ae42905c0f26fcfa9353e48e1a001b5b5c3c94b33e2ceacc6bc84c23c61a88" %}
+{% set version = "1.11" %}
+{% set sha256 = "d03208fd7ef983e4a606f861a5efcdc5a1980952a91febba3f67057800da6701" %}
+{% set runc_version = "1.0.0-rc5" %}
```

Comment:
This is how it is tagged on GitHub (with a `v` prefix, which we strip from the version). Is this the right we to handle RCs with Go?